main.go

```diff
@@ -86,10 +91,12 @@ func (configs ConfigsModel) print() {
 	log.Printf("- NotifyEmailList: %s", configs.NotifyEmailList)
 	log.Printf("- IsPublicPageEnabled: %s", configs.IsPublicPageEnabled)
 	log.Printf("- PublicInstallPageMapFormat: %s", configs.PublicInstallPageMapFormat)
+	log.Printf("- ClearDirectoryWhenFinished: %s", configs.ClearDirectoryWhenFinished)
 }
 
 func fail(format string, v ...interface{}) {
 	log.Errorf(format, v...)
+	clearDirectoryIfNeeded()
```

Comment:
Please do not add this method to failf, as the method name does not indicates the cleanup. Please create a method for example mainE where the cleanup is done and calls failf. See this as example:
https://github.com/bitrise-steplib/bitrise-step-android-build/blob/master/main.go#L328